client/src/components/assessments/InstantAssessmentResults.js

```diff
@@ -63,3 +63,85 @@ InstantAssessmentsRow.propTypes = {
   questionConfig: PropTypes.object.isRequired,
```

Comment:
Sometimes, person pages give the following warning -> Failed prop type: Invalid prop `questionConfig` of type `boolean` supplied to `InstantAssessmentsRow`, expected `object`. 
It is always shown if you navigate to a person show page from the person links inside process board. However, it also appears from time to time when you navigate from different links on different pages. Very hard to reproduce those cases.